.gitignore

```diff
@@ -40,7 +40,6 @@ coverage.xml
 
 # Translations
 *.mo
-*.pot
```

Comment:
I never read it, so I just was being guided by actual OCA code, but I'll undo it.